Presentation/src/main/java/hu/kalee/skeleton/presentation/controller/FormController.java

```diff
@@ -25,61 +25,80 @@
  * @since 2015.12.23..
  */
 @Controller
-public class FormController {
-    @Inject
-    BusinessFacade facade;
+public class FormController extends BaseController<FormDTO, BusinessInputDTO, BusinessOutputDTO, ResultDTO> {
+    public static final String CALLING_URL = "form";
+    public static final String CALLING_VIEW_NAME = "form";
+    public static final String OK_VIEW_NAME = "redirect:/result";
```

Comment:
Should contain only the view name, and integrate calling method (redirect) into base flow.